src/box/index.h

```diff
@@ -623,11 +623,36 @@ index_get(struct index *index, const char *key,
 	return index->vtab->get(index, key, part_count, result);
 }
 
+/**
+ * Get tuple to be inserted in index, based on index-specific constraints
+ * (current constraint: if exclude_null = true, return NULL)
+ * @param index
+ * @param tuple
```

Comment:
I'd better remove this uninformative doxygen comments (or add smth meaningful).

Reply:
fixed